src/Core/Grid/Definition/Factory/CarrierGridDefinitionFactory.php

```diff
@@ -0,0 +1,289 @@
+                    ->setOptions([
+                        'actions' => $this->getRowActions(),
+                    ])
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getFilters()
+    {
+        return (new FilterCollection())
+            ->add(
+                (new Filter('id_carrier', TextType::class))
+                    ->setAssociatedColumn('id_carrier')
+                    ->setTypeOptions([
+                        'attr' => [
+                            'placeholder' => $this->trans('Search ID', [], 'Admin.Actions'),
+                        ],
+                        'required' => false,
+                    ])
+            )
+            ->add(
+                (new Filter('name', TextType::class))
+                    ->setAssociatedColumn('name')
+                    ->setTypeOptions([
+                        'attr' => [
+                            'placeholder' => $this->trans('Search name', [], 'Admin.Actions'),
+                        ],
+                        'required' => false,
+                    ])
+            )
+            ->add(
+                (new Filter('delay', TextType::class))
+                    ->setAssociatedColumn('delay')
+                    ->setTypeOptions([
+                        'required' => false,
+                    ])
+            )
+            ->add(
+                (new Filter('active', YesAndNoChoiceType::class))
+                    ->setAssociatedColumn('active')
+                    ->setTypeOptions([
+                        'required' => false,
+                    ])
+            )
+            ->add(
+                (new Filter('is_free', YesAndNoChoiceType::class))
+                    ->setAssociatedColumn('is_free')
+                    ->setTypeOptions([
+                        'required' => false,
+                    ])
+            )
+            ->add(
+                (new Filter('position', TextType::class))
+                    ->setAssociatedColumn('position')
+                    ->setTypeOptions([
+                        'required' => false,
+                    ])
+            )
+            ->add(
+                (new Filter('actions', SearchAndResetType::class))
+                    ->setAssociatedColumn('actions')
+                    ->setTypeOptions([
+                        'reset_route' => 'admin_common_reset_search_by_filter_id',
+                        'reset_route_params' => [
+                            'filterId' => self::GRID_ID,
+                        ],
+                        'redirect_route' => 'admin_carriers_index',
+                    ])
+            );
+    }
+
+    protected function getGridActions()
+    {
+        return (new GridActionCollection())
+            ->add(
+                (new SimpleGridAction('common_refresh_list'))
+                    ->setName($this->trans('Refresh list', [], 'Admin.Advparameters.Feature'))
```

Comment:
Shouldn't it be located in Admin.Advparameters.Actions?

Reply:
It should have been localized in _Admin.Actions_ but, historically, it falls into _Admin.Advparameters.Feature_. I'm fine with it but if you prefer to localize it as an action wording, just note that it will have to be harmonized during each migration. :-)

Reply:
In that case, we should keep it in Admin.Advparameters.Actions. Thank you Louise !